src/main/java/xyz/nikitacartes/easyauth/event/AuthEventHandler.java

```diff
@@ -55,13 +54,39 @@ public static LiteralText checkCanPlayerJoinServer(GameProfile profile, PlayerMa
                             config.lang.playerAlreadyOnline, onlinePlayer.getName().asString()
                     )
             );
-        } else if (!matcher.matches()) {
+        }
+        if (!matcher.matches()) {
```

Comment:
oh i think it was actually a mistake lol i deleted a bunch of lines there by accident and was too lazy to put it back since they were mostly minor stuff